packages/curve-ui-kit/src/shared/ui/stories/LargeTokenInput.stories.tsx

```diff
@@ -0,0 +1,234 @@
+import { useState } from 'react'
+import { Select, MenuItem, Typography, Stack } from '@mui/material'
+import type { Meta, StoryObj } from '@storybook/react'
+import { fn } from '@storybook/test'
+import { LargeTokenInput } from '../LargeTokenInput'
+
+const TOKEN_OPTIONS = [
+  { name: 'Inferium', symbol: 'ETH', balance: 1000 },
+  { name: 'Bitcorn', symbol: 'BTC', balance: 2500 },
+  { name: 'The Ripple', symbol: 'RPL', balance: 5000 },
+]
+
+type TokenSelectorProps = {
+  onTokenChange: (tokenInfo: { balance: number; symbol: string }) => void
+}
+
+const TokenSelector = ({ onTokenChange }: TokenSelectorProps) => {
+  const [value, setValue] = useState(TOKEN_OPTIONS[0].name)
+
+  return (
+    <Select
+      value={value}
+      onChange={(e) => {
+        const selectedToken = e.target.value
+        setValue(selectedToken)
+        const selectedOption = TOKEN_OPTIONS.find((option) => option.name === selectedToken)
+
+        if (selectedOption) {
+          onTokenChange({
```

Comment:
wouldn't it be simpler to pass the whole option including the name?

Reply:
I on't think so, I only care about the sliders being configurable as args for the storybook. I want the name and symbol to come from the tokenselector itself.